vue/components/ui/molecules/lightbox/lightbox.vue

```diff
@@ -107,6 +112,12 @@ export const Lightbox = {
         close() {
             this.$emit("close");
         }
+    },
+    computed: {
+        imageStyle() {
+            const base = { "object-fit": this.objectFit };

```

Comment:
This works, but since we're doing this everywhere...
```suggestion
            const base = { "object-fit": this.objectFit };
            if (this.objectFit) base["object-fit"] = this.objectFit;
```